check_stability.py

```diff
@@ -582,6 +622,11 @@ def get_parser():
                         # This is a workaround to get what should be the same value
                         default=os.environ.get("TRAVIS_REPO_SLUG").split('/')[0],
                         help="Travis user name")
+    parser.add_argument("--output-cap",
+                        action="store",
+                        default=sys.maxint,
```

Comment:
Using `None` would force us to opt-out of argparse's type enforcement. What do you think about using `-1` to signify "no limit?"